Microsoft Defender/microsoft-defender-for-endpoints/tests/test_device_event.json

```diff
@@ -0,0 +1,87 @@
+    "event": {
+      "kind": "event",
+      "type": [
+        "info"
+      ],
+      "dataset": "device_events",
+      "category": ["host"]
+    },
+    "destination": {
+      "ip": "5.6.7.8",
+      "address": "5.6.7.8"
+    },
+    "host": {
+      "id": "1111111111111111111111111111111111111111",
+      "name": "test.lab"
+    },
+    "process": {
+      "pid": 1664,
+      "start": "2022-09-01T06:56:23.7887846Z",
+      "executable": "software_reporter_tool.exe",
+      "command_line": "\"software_reporter_tool.exe\" --use-crash-handler-with-id=\"\\\\.\\pipe\\crashpad_11111_XXXXXXXXXXXXXXXX\" --sandboxed-process-id=2 --init-done-notifier=804 --sandbox-mojo-pipe-token=********** --mojo-platform-channel-handle=780 --engine=2",
```

Comment:
Seems not usable for detection, maybe the escape chars come from your tests ?

Reply:
The escaped chars are only for the test. For the detection, you will have the string:
```
"software_reporter_tool.exe" --use-crash-handler-with-id="\\.\pipe\crashpad_11111_XXXXXXXXXXXXXXXX" --sandboxed-process-id=2 --init-done-notifier=804 --sandbox-mojo-pipe-token=********** --mojo-platform-channel-handle=780 --engine=2
```